frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/DemoGraphTable.java

```diff
@@ -12,15 +12,19 @@
 
 public final class DemoGraphTable extends JBTable {
 
-  public static final DemoGraphTable INSTANCE = deriveInstance();
+  private static final GraphTableModel graphTableModel = deriveGraphTableModel();
 
   @UIEffect
-  private static DemoGraphTable deriveInstance() {
+  public static DemoGraphTable deriveInstance() {
+    return new DemoGraphTable(graphTableModel);
```

Comment:
Maybe add a comment that/why we can retain the same repository graph but not the same graph table